Source/Csla.Blazor/ApplicationContextManager.cs

```diff
@@ -49,19 +49,14 @@ public ApplicationContextManager(AuthenticationStateProvider authenticationState
       InitializeUser();
     }
 
-    private async void InitializeUser()
+    private void InitializeUser()
```

Comment:
@TheCakeMonster I think this is what you suggested? It does seem cleaner and probably safer.

Reply:
Yes, that looks good.

I wrote it out as a separate method to make the readability as good as it could get, but the lambda works too.